as-vending/res/configuration.toml

```diff
@@ -46,18 +46,18 @@ Type="edgex-messagebus"
 
 [ApplicationSettings]
   AuthenticationEndpoint = "http://localhost:48096/authentication"
-  DeviceControllerBoarddisplayReset = "http://localhost:48082/api/v2/device/name/ds-controller-board/command/displayReset"
-  DeviceControllerBoarddisplayRow0 = "http://localhost:48082/api/v2/device/name/device-controller-board/command/displayRow0"
-  DeviceControllerBoarddisplayRow1 = "http://localhost:48082/api/v2/device/name/device-controller-board/command/displayRow1"
-  DeviceControllerBoarddisplayRow2 = "http://localhost:48082/api/v2/device/name/device-controller-board/command/displayRow2"
-  DeviceControllerBoarddisplayRow3 = "http://localhost:48082/api/v2/device/name/device-controller-board/command/displayRow3"
-  DeviceControllerBoardLock1 = "http://localhost:48082/api/v2/device/name/device-controller-board/command/lock1"
-  DeviceControllerBoardLock2 = "http://localhost:48082/api/v2/device/name/device-controller-board/command/lock2"
+  DeviceControllerBoarddisplayReset = "displayReset"
+  DeviceControllerBoarddisplayRow0 = "displayRow0"
+  DeviceControllerBoarddisplayRow1 = "displayRow1"
+  DeviceControllerBoarddisplayRow2 = "displayRow2"
+  DeviceControllerBoarddisplayRow3 = "displayRow3"
+  DeviceControllerBoardLock1 = "lock1"
+  DeviceControllerBoardLock2 = "lock2"
```

Comment:
`display` => `Display` and these need to have `Command` or `Cmd` in the names. Preferably drop `Device` from the names. i.e `ControllerBoardDisplayResetCmd`